#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_default_documenter.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dir seperator for POSIX

@DPATCH@

diff -ur ndoc-1.3.1-old/src/Console/Console.cs ndoc-1.3.1/src/Console/Console.cs
--- ndoc-1.3.1-old/src/Console/Console.cs	2005-12-17 15:53:00.000000000 +0100
+++ ndoc-1.3.1/src/Console/Console.cs	2005-12-17 15:53:30.000000000 +0100
@@ -40,7 +40,7 @@
 				WriteLogoBanner();
 
 				project = new Project();
-				documenter = project.GetDocumenter("MSDN");
+				documenter = project.GetDocumenter("LinearHtml");
 				if (documenter == null)
 				{
 					//MSDN documenter not found, pick the first one available.
